packages/logger/src/Logger.ts

```diff
@@ -558,34 +522,31 @@ class Logger extends Utility implements ClassThatLogs {
   }
 
   /**
-   * Creates a new Logger instance.
+   * Factory method for instantiating logger instances. Used by `createChild` method.
+   * Important for customization and subclassing. It allows subclasses, like `MyOwnLogger`,
+   * to override its behavior while keeping the main business logic in `createChild` intact.
    *
-   * @param {ConstructorOptions} [options]
-   * @returns {Logger}
-   */
-  protected createLogger(options?: ConstructorOptions): Logger {
-    return new Logger(options);
-  }
-
-  /**
-   * Decides whether the current log item should be printed or not.
+   * **Please do not remove this method during refactoring!**
```

Comment:
I wouldn't add this to the comments - let's just explain what this method is for & how it's used (which you did). This way future readers will see that it has an actual usage & it won't be removed.